fyle_accounting_mappings/models.py

```diff
@@ -189,16 +190,20 @@ def bulk_create_or_update_expense_attributes(
                         value=attribute['value'],
                         source_id=attribute['source_id'],
                         detail=attribute['detail'] if 'detail' in attribute else None,
-                        workspace_id=workspace_id
+                        workspace_id=workspace_id,
+                        active=attribute['active'] if 'active' in attribute else None
                     )
                 )
             else:
                 if update and 'detail' in attribute and \
-                        attribute['detail'] != primary_key_map[attribute['value']]['detail']:
+                        attribute['detail'] != primary_key_map[attribute['value']]['detail']or\
+                        ('active' in attribute and 
+                            attribute['active'] != primary_key_map[attribute['value']]['active']):
                     attributes_to_be_updated.append(
                         ExpenseAttribute(
                             id=primary_key_map[attribute['value']]['id'],
-                            detail=attribute['detail'],
+                            detail=attribute['detail'] if 'detail' in attribute else None,
+                            active=attribute['active'] if 'active' in attribute else None
                         )
                     )
         if attributes_to_be_created:
```

Comment:
in line 213 - you should update active field also 
`            ExpenseAttribute.objects.bulk_update(attributes_to_be_updated, fields=['detail', 'active'], batch_size=50)
`

Reply:
Done